grails-app/views/userProfile/basic.gsp

```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+	    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+	    <r:require module="main_mod"/>
+	    <title>User profile</title>
+	    <r:layoutResources/>
+	    
+	    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Open+Sans:400,600,700"/>
+	    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet">
+		<link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
+
+        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+
+        <style>
+            *, *:before, *:after {
+                margin: 0;
+                padding: 0;
+                box-sizing: border-box;
+            }
+
+            p {
+                margin: 0 0 20px;
+                line-height: 1.5;
+            }
+
+            section {
+                display: none;
+                padding: 20px 0 0;
+                border-top: 1px solid #ddd;
+            }
+
+            .profileTab {
+                display: none;
+            }
+
+            .tabLabel {
+                display: inline-block;
+                margin: 0 0 -1px;
+                padding: 10px 20px;
+                font-weight: 600;
+                text-align: center;
+                color: #808080;
+                border: 1px solid transparent;
+            }
+
+            .tabLabel:hover {
+                color: #888;
+                cursor: pointer;
+            }
+
+            input:checked + label {
+                color: #555;
+                border: 1px solid #ddd;
+                border-bottom: 1px solid #fff;
+            }
+
+            #tab1:checked ~ #content1,
+            #tab2:checked ~ #content2 {
+                display: block;
+            }
+        </style>
+
+	</head>
+	
+	
+	
+	<body>
+		<div id="header-div">
+		    <g:render template="/layouts/commonheader" model="['app': '']"/>
```

Comment:
no need for quotes around `app` since Groovy allow you to omit map keys when they're strings